torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -500,8 +500,6 @@ struct Environment {
           {"chr", std::make_shared<BuiltinFunction>(aten::chr, at::nullopt)},
           {"bin", std::make_shared<BuiltinFunction>(aten::bin, at::nullopt)},
           {"pow", std::make_shared<BuiltinFunction>(aten::pow, at::nullopt)},
-          {"complex",
-           std::make_shared<BuiltinFunction>(aten::complex, at::nullopt)},
```

Comment:
Why did you get rid of this? To do in a followup diff?

Reply:
this was added in https://github.com/pytorch/pytorch/pull/53227 in an effort to support complex constructor for some inputs. However, with the change above, more input variants are supported